Gemfile

```diff
@@ -39,6 +39,8 @@ gem 'uglifier', '>= 1.0.3'
 gem 'unicorn'
 gem 'validates_formatting_of'
 gem 'will_paginate'
+gem 'elasticsearch-model', '~> 0.1.7'
+gem 'elasticsearch-rails', '~> 0.1.7'
```

Comment:
Yep, we're implementing that in https://github.com/rubygems/rubygems.org/compare/search2 in fact.